### PR TITLE
[enterprise-4.7] Update to the Insights Operator RN

### PR DESCRIPTION
**Pull request by xJustin**
This is an update to the RN for 4.7
Removed
https://bugzilla.redhat.com/show_bug.cgi?id=1893953
Added
https://bugzilla.redhat.com/show_bug.cgi?id=1921013
https://bugzilla.redhat.com/show_bug.cgi?id=1917799

@mmuehlfeldRH can you please review

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -287,7 +287,8 @@ In {product-title} 4.7, the Insights Operator collects the following additional
 * The `ContainerRuntimeConfig` and `MachineConfigPools` configuration files to verify container storage limits
 * The configuration files for all available `operator.openshift.io` control pane resources to identify Operators in unmanaged states
 * The `NetNamespaces` names, including their `netID` and egress IP addresses
-* The `StatefulSet` configuration definitions from the {product-title} default namespaces to check the Prometheus monitoring storage type
+* A list of all installed OLM Operators, including version information
```
Comment (mmuehlfeldRH) [inline release_notes/ocp-4-7-release-notes.adoc:290]: Do we need to expand OLM?

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -287,7 +287,8 @@ In {product-title} 4.7, the Insights Operator collects the following additional
...
+* The Persistent Volume definition, if used in the Image Registry Operator configuration
```
Comment (mmuehlfeldRH) [inline release_notes/ocp-4-7-release-notes.adoc:291]: Does "Persistent Volume" need some markup?
Same question for "Image Registry"?

File: release_notes/ocp-4-7-release-notes.adoc (hunk 1 above)
Comment (xJustin) [reply release_notes/ocp-4-7-release-notes.adoc]: expanded

File: release_notes/ocp-4-7-release-notes.adoc (hunk 2 above)
Comment (xJustin) [reply release_notes/ocp-4-7-release-notes.adoc]: I don't think it's appropriate for Persistent Volume, but I changed Image Registry to `openshift-image-registry` namespace

**Comment by xJustin** [comment]
@openshift/team-documentation can you please review this. Thanks

**Comment by ahardin-rh** [comment]
This looks good to me. Thank you!

**Pull request merged**